TLS_E2E_Automation_Framework/cypress/e2e/tls_page_elements/myAccount.js

```diff
@@ -0,0 +1,105 @@
+const myAccountLink = "//li[@class='nk-menu-item css-bkfzpe']";
+const homePageLink = "//div[@class='css-mycqxl']//span[@class='css-1ayf1st']";
+const subsDetail = ".css-ztoeyf";
+const subText = "You don’t have a subscription";
+const addressLink = "./css-13p0b4b";
+const footerLink = "//div[@class='css-1c1zire']/a";
+const goToLink = ".css-mycqxl";
+const homaPageLink = ".tls-card-headline";
+const myAccountLinkHome =
+  "//*[@id='tls-home-page-root']/div/div/nav/div/div[2]/span[5]/a";
+const subscribeButton = ".tls-header-navigation__button-subscribe > a";
+
+export function myAccountLinkValidation() {
+  cy.xpath(myAccountLink).eq(0).should("exist");
+  cy.xpath(myAccountLink).eq(1).should("exist");
```

Comment:
please check here also, do we have class or id's, we can make use of it

Reply:
same like last usage